src/ring/middleware/defaults.clj

```diff
@@ -78,7 +77,9 @@
         args))
 
 (defn- wrap-xss-protection [handler options]
-  (x/wrap-xss-protection handler (:enable? options true) (dissoc options :enable?)))
+  (x/wrap-xss-protection handler
+                         (:enable? options false)
+                         (-> options (dissoc :enable?) not-empty)))
```

Comment:
Are these changes still necessary? What happens if we take them out?

Reply:
Hard to say, at least unit tests still pass.